client/app/components/Modal/index.jsx

```diff
@@ -102,8 +102,8 @@ export const Modal = (props: Props): Node => {
     <div
       className={`modalBackdrop ${css.modalBackdrop}`}
       onClick={handleClick}
-      onKeyDown={handleKeyPress}
-      tabIndex="0"
+      onKeyDown={(event) => handleKeyPress(event, 'Escape')}
+      tabIndex="-1"
```

Comment:
We do not want to be able to focus on elements behind the modal when it's opened.

![Oct-02-2022 00-08-23](https://user-images.githubusercontent.com/3010728/193437748-43aabb01-f027-4e9d-b317-bdb5710d8b8e.gif)